build/UIKit/lib/UIKitLib.vcxproj

```diff
@@ -110,6 +110,20 @@
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIView.mm" />
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIViewController.mm" />
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIWindow.mm" />
+    <ClangCompile Include="..\..\..\Frameworks\UIKit\WYPopoverController.m">
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">true</ObjectiveCARC>
+      <OptimizationLevel Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">Disabled</OptimizationLevel>
+      <OptimizationLevel Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">Disabled</OptimizationLevel>
```

Comment:
@oliversa-msft It is to address the preferredContentSize issue. I'd have to do a clean framework build to confirm but I'm pretty sure you'll see an undefined (== fullscreen) height if you pref optimization back on.

In general WYPopoverController is pretty gross in that it swizzles all calls to preferredContentSize in order to modify it's internal state. A _from scratch_ popover implementation could instead handle this with some extra logic in UIViewController.mm